tests/storage/datavolume.go

```diff
@@ -501,7 +563,12 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 		BeforeEach(func() {
 			running := true
 
-			vm = tests.NewRandomVMWithDataVolume(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), util.NamespaceTestDefault)
+			var foundCS bool
+			vm, foundCS = tests.NewRandomVMWithDataVolume(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), util.NamespaceTestDefault)
```

Comment:
Can you use `:=` here? And name it `foundSC` (typo?)

Reply:
can't use `:=` - it will override the `vm` variable.

Fixed typo.